internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1647,14 +1660,7 @@ func (r *ClusterDeploymentsReconciler) handleClusterInstalled(ctx context.Contex
 		}
 		return r.updateStatus(ctx, log, clusterInstall, cluster, err)
 	} else if r.EnableDay2Cluster && !r.isSNO(clusterInstall) {
```

Comment:
Will you remove `EnableDay2Cluster` flag in a different PR? we don't need it if we are using the same cluster in the backend.

Reply:
As we discussed, will do this in a follow-up PR.